scm/driver/github/util.go

```diff
@@ -29,7 +29,8 @@ func encodeRepoListOptions(opts scm.RepoListOptions) string {
 		if opts.RepoSearchTerm.RepoName != "" {
 			sb.WriteString("q=")
 			sb.WriteString(opts.RepoSearchTerm.RepoName)
-			sb.WriteString("in:name+user:")
+			// %20 is urlEncoding of blank space.
+			sb.WriteString("%20in:name+user:")
```

Comment:
can we use `url.Values` to encode instead of string builder?

Reply:
@bradrydzewski  
We have to use the string builder here as we need to make manually make a string for query at Github Side. Here we are not adding query params in the API. 

Here's the link for reference in the API we are using : https://docs.github.com/en/rest/search/search?apiVersion=2022-11-28.
Thanks for the insightful review.